gradle-multi-module/sample-admin/src/main/java/com/sungjun/admin/rest/AppController.java

```diff
@@ -0,0 +1,86 @@
+package com.sungjun.admin.rest;
+
+import com.sungjun.bank.api.BankApi;
+import com.sungjun.cloud.service.SubscriptionApi;
+import com.sungjun.cloud.service.UserService;
+import com.sungjun.dto.entity.BankCard;
+import com.sungjun.dto.entity.BankCardType;
+import com.sungjun.dto.entity.Subscription;
+import com.sungjun.dto.entity.UserEntity;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.data.crossstore.ChangeSetPersister.NotFoundException;
```

Comment:
Can we use custom exception here?

---

21. Create Exception class which will be used into orElseThrow for Optional from getSubscriptionByBankCardNumber().